swagger/src/main/java/org/scalatra/swagger/runtime/annotations/ApiModelProperty.java

```diff
@@ -47,7 +47,7 @@
     * allows explicitly ordering the property in the model.  Since reflection has no guarantee on
     * ordering, you should specify property order to keep models consistent across different VM implementations and versions.
     */
-    int position() default 0;
+    int position() default Integer.MAX_VALUE;
```

Comment:
how do you feel about using java.lang.Integer/null instead?

Reply:
Annotations can't be null https://stackoverflow.com/questions/24681223/annotation-default-null-value

Reply:
ah so...